pkg/volume/iscsi/iscsi_util.go

```diff
@@ -370,6 +386,16 @@ func (util *ISCSIUtil) DetachDisk(c iscsiDiskUnmounter, mntPath string) error {
 					glog.Errorf("iscsi: failed to delete node record Error: %s", string(out))
 				}
 			}
+			// Delete the iface after all sessions have logged out
+			// If the iface is not created via iscsi plugin, skip to delete
+			if initiatorName != "" && found && iface == (portals[0]+":"+c.volName) {
```

Comment:
if loaded from json, portals has the possibility to be zero-length, and this will panic